### PR TITLE
docs: Add dual license

### PR DESCRIPTION
**Pull request by RichardLitt**
Missing it in the manifest. Not sure if this is how GX reads it (cc @whyrusleeping), but I'm going off of [this example from @kemitchell](https://www.npmjs.com/package/dual-licensed-package).

### CODE REVIEW

**Comment by phritz** [comment]
@dignifiedquire

File: package.json [hunk 1]
```diff
@@ -198,7 +198,7 @@
   ],
   "gxVersion": "0.12.1",
   "language": "go",
-  "license": "",
+  "license": "(MIT OR APACHE)",
```
Comment (dignifiedquire) [inline package.json]: should be `Apache 2` I believe

Comment (RichardLitt) [reply package.json]: https://spdx.org/licenses/ says Apache-2.0. Changed!

Comment (dignifiedquire) [reply package.json]: perfect, thank you

**Pull request merged**